src/pages/[community]/index.tsx

```diff
@@ -12,30 +11,28 @@ type Props = {
 }
 
 const CommunityPage = ({ communityData }: Props): JSX.Element => (
-  <CommunityProvider data={communityData}>
```

Comment:
I think we used the `CommunityProvider` to display the color gradient background for the community page (e.g. check the difference between the `/agora/` and `/agora/community` pages), so it would be great if we could keep it.

Reply:
You are absolutely right, I forgot about that! I didn't find any `useCommunity` calls, and didn't find any, that's why I thought its unnecessary.